include/aws/s3/s3_endpoint_resolver/s3_endpoint_resolver.h

```diff
@@ -0,0 +1,49 @@
+#ifndef AWS_S3_ENDPOINT_RESOLVER_H
+#define AWS_S3_ENDPOINT_RESOLVER_H
+
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+#include <aws/s3/s3.h>
+#include <aws/sdkutils/endpoints_rule_engine.h>
+
+AWS_PUSH_SANE_WARNING_LEVEL
+extern const char aws_s3_endpoint_resolver_partitions[];
```

Comment:
do we need it customer header? or can we have it in private header?

Reply:
Thanks, fixed.